rmc/resources/grch37/rmc.py

```diff
@@ -0,0 +1,374 @@
+"""
+Script containing RMC and MPC related resources.
+
+RMC: Regional missense constraint
+MPC: Missense badness, Polyphen-2, and Constraint score
+"""
+import hail as hl
+
+from gnomad.resources.resource_utils import TableResource, VersionedTableResource
+
+from rmc.resources.basics import (
+    CONSTRAINT_PREFIX,
+    MODEL_PREFIX,
+    MPC_PREFIX,
+    RESOURCE_PREFIX,
+)
+from rmc.resources.resource_utils import CURRENT_VERSION
+
+
+FREEZES = [1, 2]
```

Comment:
Do we want to make the freeze names also tell you which gnomAD version they correspond to?

Reply:
So is the idea that freeze numbers for RMC/MPC calculated on gnomAD v4 will continue where the numbers for gnomAD v2 leave off? i.e. if freeze 5 is the final RMC/MPC freeze for gnomAD v2, then freeze 6 will be the first for v4? I guess this would make ExAC RMC/MPC freeze 0.

Reply:
hmmmm I hadn't thought about this in much detail, but I like this idea (continuing to increment on freeze number)